client/payment-methods/index.js

```diff
@@ -55,8 +55,8 @@ const availableMethods = [
 ];
 
 const PaymentMethods = ( { enabledMethodIds, onEnabledMethodIdsChange } ) => {
-	const enabledMethods = availableMethods.filter( ( method ) =>
-		enabledMethodIds.includes( method.id )
+	const enabledMethods = enabledMethodIds.map( ( methodId ) =>
```

Comment:
We need to keep the original sorting of `enabledMethodIds`, not the sorting of `availableMethod`. For this reason, I changed the `.map` to map on `enabledMethodIds` instead.